Sources/XiEditor/ClientImplementation.swift

```diff
@@ -182,30 +179,34 @@ class ClientImplementation: XiClient, DocumentsProviding, ConfigCacheProviding,
 
     // Stores the config dict so new windows don't have to wait for core to send it.
     // The main purpose of this is ensuring that `unified_titlebar` applies immediately.
-    var configCache: [String: AnyObject] = [:]
+    var configCache = Config(fontFace: nil, fontSize: nil, scrollPastEnd: nil, unifiedToolbar: nil)
```

Comment:
The cache is now a strongly-typed object instead of a bucket of key-value pairs. They're all nillable which I think is ok because the places we use this cache we have fallback values.